src/main/java/com/github/nicholasmoser/gnt4/seq/comment/Comments.java

```diff
@@ -25,178 +29,190 @@ public class Comments {
    * @param fileName The name of the SEQ file to get comments for.
    * @return The SEQ file offset to comment mapping.
    */
-  public static Map<Integer, String> getComments(String fileName) {
+  public static Multimap<Integer, String> getComments(String fileName) {
     if (COMMENTS == null) {
       COMMENTS = getAllComments();
     }
-    Map<Integer, String> comments =  COMMENTS.get(fileName);
+    Multimap<Integer, String> comments =  COMMENTS.get(fileName);
     if (comments == null) {
       LOGGER.log(Level.INFO, "No comments found for file " + fileName);
-      return Collections.emptyMap();
+      return ArrayListMultimap.create();
     }
     return comments;
   }
 
-  private static Map<String, Map<Integer, String>> getAllComments() {
-    Map<String, Map<Integer, String>> allComments = new HashMap<>();
+  private static Map<String, Multimap<Integer, String>> getAllComments() {
+    Map<String, Multimap<Integer, String>> allComments = new HashMap<>();
 
-    Map<Integer, String> ank0000Comments = new HashMap<>();
+    Multimap<Integer, String> ank0000Comments = ArrayListMultimap.create();
     ank0000Comments.put(0x5D50, "// Scale chr model larger by 1.1x");
```

Comment:
Is the function to scale the chr model the full 
```
130B203F 00000003 0303003F 3F8CCCCD 0303003F 
3F8CCCCD 0303003F 3F8CCCCD 0B0A006C 00000000 
090A006C 00000000 0C095300 00000074 53000000
00000074 040E5300 00000000 3F000000 000001E0 01450000
```
or is it just the 130B203F?
In the latter case, wouldn't it be better to add the info to the opcode instead?

Reply:
It's the full function. Might be good to add some kind of comment syntax here to indicate it is function documentation. Let me see how assembly code usually documents functions...

Reply:
`;` seems pretty popular for general comments. I've also seen `#` and `/* */`. Maybe it's worth just adding a function name to make it clear?

Reply:
Maybe something like this?
![image](https://user-images.githubusercontent.com/4983605/167325538-0ca29f5b-4c19-4fc7-8bdf-0ecf81a03db4.png)
https://github.com/NicholasMoser/GNTool/pull/97/commits/09a5c5ec1a473b2c9bce9e294858bb7907e49148